src/main/java/org/opensearch/security/ssl/SslContextHandler.java

```diff
@@ -74,7 +74,9 @@ void reloadSslContext() throws CertificateException {
         if (sameCertificates(newCertificates)) {
             return;
         }
-        validateNewCertificates(newCertificates);
+        if (sslConfiguration.sslParameters().isValidateCertsOnReloadEnabled()) {
```

Comment:
The code needs to be moved inside `validateNewCertificates`. We need to validate notAfter and notBefore  oin any case

Reply:
Moved the logic to after https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/ssl/SslContextHandler.java#L146